### ISSUE TITLE
FTX watch_orders fails

### ISSUE THREAD

**Issue by diwenshi61** [ccxt/ccxt#7855]
For FTX I haven't been able to get the watch_orders method to work.

- OS: Windows
- Programming Language version: Python 3.8.5
- CCXT version: 0.4.39

```
import ccxtpro
import asyncio

async def main():
    binance =  ccxtpro.ftx({
        'apiKey': api_key_here,
        'secret': api_secret_here,
        'headers': {
            'FTX-SUBACCOUNT': subaccount_name_here,
        },
        'enableRateLimit': False
    })
    while True:
        orderbook = await binance.watch_orders()
        print(orderbook)

asyncio.get_event_loop().run_until_complete(main())
```

```
  File "C:\.........\Python38\site-packages\ccxtpro\ftx.py", line ..., in authenticate
    future.resolve(True)
  AttributeError: 'coroutine' object has no attribute 'resolve'
```
I thought it may have something to do with authenticate not being defined as an async procedure, but I haven't looked at it.

**Comment by kroitor** [comment]
Thank you for reporting it! We have received other reports of the same issue, and we are now working on the fix. It should arrive before the end of the week.

**Comment by kroitor** [comment]
@diwenshi61 fixed by @frosty00 in version 0.4.40+, the new build will be available in 20 minutes. Thx for your feedback!
